openfga/v1/authzmodel.proto

```diff
@@ -10,7 +10,7 @@ message AuthorizationModel {
   string id = 1 [
     (google.api.field_behavior) = REQUIRED,
     (validate.rules).string = {
-      pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$",
+      pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$"
```

Comment:
**Non-blocking question:** Do you know why all these comma removal lines were generated?

Reply:
I've been trying to track that down, this came from me running `make` (specifically `buf format -w`) and also seems to occur on `main` also.

I'm guessing it's down to a change in buf format but I haven't got as far as looking what changed

Reply:
Turned out this was due to a change in buf v1.33.0, specifically https://github.com/bufbuild/buf/commit/c0972ddf4a9a6ba12768c53a2f7ef55759a4ea53

I filed #181 to move away from pinning the buf version used so that we can keep up to date on the version of that.